proto/launch/tx.proto

```diff
@@ -116,7 +116,7 @@ message MsgRequestRemoveValidatorResponse {
 }
 
 message MsgSettleRequest {
-  string coordinator = 1;
+  string settler = 1;
```

Comment:
```suggestion
  string signer = 1;
```

Although we actually must rename this field, `settler` means more someone who migrates to a new area

Reply:
Tricky english =)